.github/label-pr-config.yml

```diff
@@ -0,0 +1,83 @@
+autolabeler:
+  - label: 'Type: documentation'
+    files:
+      - '*.md'
+    # Checks there is no word character (alphanumeric & underscore) before target words, or after to avoid accidental matches
+    title:
+      - '/(?<![\w])((doc)|(docs)|(documentation))(?![\w])/i'
+  - label: 'Type: fix'
+    branch:
+      - '/fix\/.+/'
+    title:
+      - '/(?<![\w])((fix)|(fixes)|(fixed)|(bug)|(bugfix)|(bug fix))(?![\w])/i'
+  - label: 'Type: feature'
+    branch:
+      - '/feature\/.+/'
+    title:
+      - '/(?<![\w])((feat)|(feature))(?![\w])/i'
+  - label: 'Type: enhancement'
+    branch:
+      - '/enhancement\/.+/'
+    title:
+      - '/(?<![\w])((enhancement)|(enhancements))(?![\w])/i'
+  - label: 'Type: buildscripts'
+    files:
+      - '*.yml'
+    branch:
+      - '/buildscripts\/.+/'
+    title:
+      - '/(?<![\w])((buildscript)|(buildscripts))(?![\w])/i'
+
+# Config file for release drafter; the labeler creates a draft release but does not publish
+name-template: 'v$RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
+categories:
+  - title: 'Features'
+    labels: 
+      - 'Type: feature'
+  - title: 'Enhancements'
+    labels: 
+      - 'Type: enhancement'
+  - title: 'Bug Fixes'
+    labels:
+      - 'Type: bugfix'
+  - title: 'Deprecations'
+    labels:
+      - 'Type: deprecated'
+  - title: 'Maintenance'
+    labels: 
+      - 'Type: buildscripts'
+      - 'Type: documentation'
+      - 'Type: maintenance'
+include-labels:
+  - 'Type: bug'
+  - 'Type: bugfix'
+  - 'Type: buildscripts'
+  - 'Type: deprecated'
+  - 'Type: documentation'
+  - 'Type: enhancement'
+  - 'Type: feature'
+  - 'Type: fix'
+  - 'Type: maintenance'
+
+change-template: '$TITLE (#$NUMBER) @$AUTHOR'
+change-title-escapes: '\<*_&' # You can add # and @ to disable mentions, and add ` to disable code blocks.
+version-resolver:
+  major:
+    labels:
+      - 'major'
+  minor:
+    labels:
+      - 'minor'
+  patch:
+    labels:
+      - 'patch'
+  default: patch
+template: |
+  ## Changes
+
+  $CHANGES
+
+  **Full Changelog**: https://github.com/$OWNER/$REPOSITORY/compare/$PREVIOUS_TAG...$RESOLVED_VERSION
```

Comment:
is this config required for the autolabeler or can we keep this just in the release-drafter?

Reply:
I couldn't figure out how to prevent the labeler config from generating a changelog, so I was thinking if we copy the exact config from the actual release drafter we can get a preview of what the changelog will look like after the labels have been applied; the labeler job is hardcoded to only create draft releases so we don't have to worry about an accidental release

What do you think?

Reply:
Ok, I think we can give few runs with current setup and test if the release drafter overwrites the labeler draft, and in case multiple drafts are created we can revisit this.